desci-server/src/middleware/permissions.ts

```diff
@@ -33,33 +33,48 @@ export const ensureUser = async (req: ExpressRequest, res: Response, next: NextF
  * Extract JWT Authorisation token from IncommingRequest
  */
 export const extractAuthToken = async (request: ExpressRequest | Request) => {
-  let token: string | undefined;
-  // get from query string
-  token = request.url.split('auth=')[1];
-  // logger.info({ url: request.url, token }, 'got url extract');
+  let token = await extractTokenFromCookie(request, 'auth');
+
   if (!token) {
-    // Try to retrieve the token from the auth header
+    // Try to retrieve the token from the header
     const authHeader = request.headers['authorization'];
     if (authHeader) {
       token = authHeader.split(' ')[1];
     }
-    logger.info({ module: 'Permissions::extractAuthToken', authHeaderLength: authHeader?.length || 0 }, 'Request');
+    logger.info({ module: 'Permissions::extractToken', authHeaderLength: authHeader?.length || 0, token }, 'Request');
```

Comment:
don't log token, this leaks info into the logs that can caused pwnage if the log service is breached